Runtime/Scripts/msCoreAPI.cs

```diff
@@ -517,15 +523,35 @@ public struct TextureRecord {
     }
 
     public string name {
-        get { return Misc.S(msMaterialPropGetName(self)); }
+        get {
+            if (self == IntPtr.Zero) {
```

Comment:
This zero check is a simple change that can be made into a different PR and merged quickly.

Reply:
Done here: https://github.com/unity3d-jp/MeshSync/pull/815/files